libgssapi/src/context.rs

```diff
@@ -747,6 +774,17 @@ impl ClientCtx {
             }
         }
     }
+
+    /// Set the channel binding token which will be incorporated
+    /// into the next generated GSSAPI token. Channel bindings are opaque
+    /// structures whose content is dictated by the application.
+    ///
+    /// The channel binding token is "spent" after the `step()` call
+    /// on the context. If you need it for another step, you must
+    /// set it again.
+    pub fn set_cb_token(&mut self, cbt: Vec<u8>) {
```

Comment:
Don't force this to be a vec. When you move it to an argument of step make it a Option<&[u8]>

Reply:
Looks good